### ISSUE TITLE
Wind direction may be incorrect

### ISSUE THREAD

**Issue by bglarkin** [samsoe/mpg_weather_warehouse#54]
@samsoe I noticed something in the app that is concerning. On the wind direction plots, the point density is strong in the south direction. This is what you'd expect with naive averaging of wind degrees. I downloaded the raw wind data from the application, and indeed it is heavily weighted to the south for every station, as seen on these histograms:

![image](https://user-images.githubusercontent.com/55502481/112384490-1a68a680-8cb4-11eb-878f-07d9d3e393ee.png)

I then downloaded a frequency table from BQ and it doesn't match:

![image](https://user-images.githubusercontent.com/55502481/112384747-6582b980-8cb4-11eb-8a66-76fa02efda8a.png)

I fear there is something wrong with the calculation of wind direction in the cloud function? Or somewhere else?

**Comment by bglarkin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Check wind direction](https://trello.com/c/0mwxC6te/245-check-wind-direction)

**Comment by samsoe** [comment]
@bglarkin the SQL in the Cloud Function is very simple
```
SELECT *
FROM `mpg-data-warehouse.weather_summaries.mpg_ranch_daily`
WHERE wspd_max_mph < 100
```

**Comment by samsoe** [comment]
> I fear there is something wrong with the calculation of wind direction in the cloud function? Or somewhere else?

@bglarkin  For clarity, I think here you are referring to how the View is constructed with SQL in BigQuery? 

Did you download the frequency table from ```weather_summaries.mpg_ranch_daily```?

**Comment by samsoe** [comment]
Wind Direction EDA Notebook here: [Link](https://colab.research.google.com/drive/1DLpr8P-qWQ5egv6eRUMdjkEPzXFFft2P?usp=sharing)

**Comment by bglarkin** [comment]
> Did you download the frequency table from `weather_summaries.mpg_ranch_daily`?

@samsoe no. Should have annotated this issue better. For the second facetted histogram plot that started this issue, I used SQL to create a frequency table from the `weatherlink` table in BQ. Counts of `Wind_Direction` grouped by `station` and `Wind_Direction`. I copied the frequency distribution and pasted to a local sandbox R project to quickly create the plots. 

What this says to me is that the raw weatherlink data shows wind direction all over the place, but somewhere along the pipeline, the vector averaging is failing and we end up with averages close to south. It could be that an incorrect variable is chosen somewhere, or something like that. The strong bias toward the south direction is almost certainly resulting from averaging untransformed degrees.
